src/main/java/org/jsoup/nodes/Entities.java

```diff
@@ -161,28 +170,30 @@ public static String escape(String string) {
             DefaultOutput = new OutputSettings();
         return escape(string, DefaultOutput);
     }
-    private static @Nullable OutputSettings DefaultOutput; // lazy-init, to break circular dependency with OutputSettings
 
-    // this method does a lot, but other breakups cause rescanning and stringbuilder generations
-    static void escape(Appendable accum, String string, OutputSettings out,
-                       boolean inAttribute, boolean normaliseWhite, boolean stripLeadingWhite, boolean trimTrailing) throws IOException {
+    // lazy-init, to break circular dependency with OutputSettings
+    @Nullable
+    private static OutputSettings DefaultOutput;
 
+    // this method does a lot, but other breakups cause rescanning and stringbuilder generations
+    static void escape(Appendable accum, String string, OutputSettings out, boolean inAttribute, boolean normaliseWhite, boolean stripLeadingWhite, boolean trimTrailing) throws IOException {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 20. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.